_docs/setup/install-kubernetes.md

```diff
@@ -73,59 +71,29 @@ default namespace. They can be modified for deployment in a different namespace.
-   
-   * If the command displays 'beta' version, or both 'alpha' and 'beta', please apply istio-rbac-beta.yaml configuration as show below:
-   
-   *(Note: If you deploy Istio in another namespace than the `default` namespace, replace the `namespace: default` line 
-   in all ClusterRoleBinding resources with the actual namespace.)*
-   
-   ```bash
-   kubectl apply -f install/kubernetes/istio-rbac-beta.yaml
-   ```
-   If you get an error
-   ```
-   Error from server (Forbidden): error when creating "install/kubernetes/istio-rbac-beta.yaml": clusterroles.rbac.authorization.k8s.io "istio-pilot" is forbidden: attempt to grant extra privileges: [{[*] [istio.io] [istioconfigs] [] []} {[*] [istio.io] [istioconfigs.istio.io] [] []} {[*] [extensions] [thirdpartyresources] [] []} {[*] [extensions] [thirdpartyresources.extensions] [] []} {[*] [extensions] [ingresses] [] []} {[*] [] [configmaps] [] []} {[*] [] [endpoints] [] []} {[*] [] [pods] [] []} {[*] [] [services] [] []}] user=&{user@example.org [...]
-   ```
-   You need to add the following: (replace the name with your own)
-   ```
-   kubectl create clusterrolebinding myname-cluster-admin-binding --clusterrole=cluster-admin --user=myname@example.org
-   ```
-   
-   * If the command displays only 'alpha' version, please apply istio-rbac-alpha.yaml configuration:
-   
-   *(Note: If you deploy Istio in another namespace than the `default` namespace, replace the `namespace: default` line 
-      in all ClusterRoleBinding resources with the actual namespace.)*
-      
-   ```bash
-   kubectl apply -f install/kubernetes/istio-rbac-alpha.yaml
-   ```
-
-1. Install Istio's core components .
-   There are two mutually exclusive options at this stage:
-
-    * Install Istio without enabling [Istio Auth]({{home}}/docs/concepts/network-and-auth/auth.html) feature:
+    * Install Istio without enabling [authentication]({{home}}/docs/concepts/network-and-auth/auth.html) between proxies with [mTLS](https://en.wikipedia.org/wiki/Mutual_authentication).
+    We recommend this option for clusters with existing applications, to allow new applications to communicate with the old applications,
+    and for clusters that use [liveliness and readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/):
 
       ```bash
       kubectl apply -f install/kubernetes/istio.yaml
       ```
-   
-      This command will install Pilot, Mixer, Ingress-Controller, Egress-Controller core components.
 
-   * Install Istio and enable [Istio Auth]({{home}}/docs/concepts/network-and-auth/auth.html) feature
-   (This deploys a CA in the namespace and enables
-   [mTLS](https://en.wikipedia.org/wiki/Mutual_authentication) between the services):
+   * Install Istio and enable [authentication]({{home}}/docs/concepts/network-and-auth/auth.html) between proxies with [mTLS](https://en.wikipedia.org/wiki/Mutual_authentication):
```

Comment:
mutual TLS authentication
between sidecars

Reply:
I wonder if the two options can be made into (a) or (b).. instead of *, you could use a., and a "OR b."

Reply:
hmm, not sure what is the syntax for this.

Reply:
I am reluctant to put that first, given the caveats.